app/controllers/visit_requests_controller.rb

```diff
@@ -32,4 +32,8 @@ def event
   def visit_request
     @visit_request ||= VisitRequest.find(params[:id])
   end
+
+  def user_verification_status
+    current_user.verified? ? :success_for_verified : :success_for_newbies
```

Comment:
I suggest using interactors. Something between controlelr features (params, flash, status and redirects) and business logic. At lead app/interactors folder already created